scripts/hooks/pre-push

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# Reject pushes that contain commits with messages that do not adhere
+# to the defined regex.
+
+# This can be a useful pre-receive hook [1] if you want to ensure every
+# commit is associated with a ticket ID.
+#
+# As an example this hook ensures that the commit message contains a
+# JIRA issue formatted as [JIRA-<issue number>].
+#
+# [1] https://help.github.com/en/enterprise/user/articles/working-with-pre-receive-hooks
+#
+
+set -e
+
+msg_regex='^[a-z]+\-[0-9]+\s|^Merge'
```

Comment:
This will break all dependable pull requests which have just been introduced to automate the manual CVE fix exercise. 

Example pull request with example commit message: https://github.com/hortonworks/cloudbreak/pull/13890

Reply:
Yes, I've tough of that. 
It should only case issues if the bot enables the .gitconfig. As mentioned in the Readme you still need to apply it to your local settings. Later on the enforcement would be through the buildInfo gralde task which I don't think the bot executes either.

Of course if you insist I can add the `^Bumps.*from` part like I did with merge.

Also feel free to come up with other exceptional cases where we don't want jira ticket ids. I've did a sweep on git log, but haven't found any valid cases.

Reply:
I see, dependabot sends a standard pull request, whatever our jenkins and RE jenkins executes will be executed. So if our standard build executes the buildInfo gradle task then it will be executed for dependabot pulls as well.

Reply:
So I think that eventually, we will need this ^Bumps.*from regexp, therefore it is better to add it now.